### DISCUSSION TITLE
请问预测总分是理论最高技能总分+属性点数吗？

### DISCUSSION THREAD

**Discussion by mtt-konan** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#57] [Q&A]
![aft2](https://user-images.githubusercontent.com/102540842/183274686-f8592933-b834-420a-85f1-feacf076a77c.jpg)

遇到了一个自己点技能分数反而高于预测总分的。正常按照列表去点技能分数是一模一样的，证明点数计算是没问题的，会不会是理论技能点最高分数计算错误了。这个问题可复现，但是我是随便点的，并没有按照其他计算技能点最高程序去计算，所以复现起来会比较困难，带有随机性。

**Comment by EtherealAO** [comment]
https://github.com/UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer/issues/5#issuecomment-1125612162
在没有点技能的时候是准的，点了就不是了，由于凹分的时候从来不会提前点技能就没管

**Reply by mtt-konan** [reply]
这个计算是提前计算的吗，所以对于提前点技能会算不准
